docs/tools/fastmodels/fastmodels.md

```diff
@@ -0,0 +1,68 @@
+## Fast Models
+
+Arm Fast Models are software models of Arm cores, subsystems, peripherals and so on. You can run Mbed OS on the software model instead of the hardware.
+
+This document explains how to build, run and debug Arm Mbed OS applications with Arm Fast Models. Due to licensing, Mbed OS do not provide any Fast Models. If you do not have a Fast Models license yet, you can visit [Arm Fast Models](https://developer.arm.com/products/system-design/fast-models) to obtain an evaluation license.
+
+Fixed Virtual Platforms (FVPs) are prebuilt system-level models after Arm’s reference platforms by Fast Models.
+
+For more details on Fast Models and FVPs, please reference further documentation about [Arm Fast Models](https://developer.arm.com/products/system-design/fast-models) or [FVPs](https://developer.arm.com/products/system-design/fixed-virtual-platforms).
+
+### Supported Fast Models FVPs
```

Comment:
This list is not going to to be updated frequently. It will get update some point next year, based on the plan. Then it will stay like that for a very long time. 
But I am open to any suggestions. Where do you think is the best place to present such information? By saying "link to somewhere", do you mean a new page in the docs? 
If we decided to keep it here, I will keep this list updated.